docs/modules/ROOT/pages/how-tos/using-customizing-policies.adoc

```diff
@@ -0,0 +1,124 @@
+= Using and Customizing Policies
+
+By installing this component default policies are created in every namespace.
+These policies are designed to be secure by default, while still allowing essential communication for most applications.
+However, depending on your specific use case, you might want to customize these policies to better fit your application's needs.
+
+== Understanding Default Policies
+
+The default policies created by this component typically include:
+
+* `base` policies: Allow essential traffic such as kubelet readiness probes.
+* `default` policies: Allow all traffic between pods within the same namespace.
+
+The `base` policy is calculated based on the keys in `basePolicy` and can't be fully customized.
+The `default` policy can be customized like every other policy using the `policies` and `policySet` keys in the component configuration.
+
+== Applying Custom Policies to a Namespace
+
+Every non-ignored namespace has the `base` and `default` policies applied by default.
+
+Custom policy sets can be applied by labeling the namespace with a `set.network-policies.syn.tools/<policy-set-name>: ""` label.
```

Comment:
Would be good to explicitly document behavior of label values other than `""` here.